services/minespace-web/src/components/dashboard/mine/permits/PermitsTable.js

```diff
@@ -97,13 +115,67 @@ export const PermitsTable = (props) => {
         width: 180,
       },
       { title: "Date Issued", dataIndex: "dateIssued", key: "dateIssued" },
+      {
+        title: "Authorization End Date",
+        dataIndex: "authorizationEndDate",
+        key: "authorizationEndDate",
+        width: "250px",
+        render: (text) => <div title="Authorization End Date">{text}</div>,
+      },
       { title: "Description", dataIndex: "description", key: "description" },
       {
-        title: "Documents",
+        title: "Map Files",
+        dataIndex: "maps",
+        key: "maps",
+        render: (text) => {
```

Comment:
is Text the required variable name here? if not can it be a little more accurate to what the data is?

Reply:
'maps' is the variable name, but all of our table renders have `(text, record)` as their positional parameters - just following the naming conventions of ant design, 'text === maps' in the data set

Reply:
Fair, I get it.